packages/Panel/src/Panel.js

```diff
@@ -177,6 +191,7 @@ export default function Panel(props) {
 
 Panel.types = {
   kind: types.sidePanelKinds,
+  slide: types.slide,
```

Comment:
this should be `slideFrom: types.slide` it should be the same name of the prop that it's representing.